exchange_server/README.md

```diff
@@ -13,8 +13,33 @@ The Exchange check is included in the [Datadog Agent][1] package, so you don't n
 
 ### Configuration
 
-1. Edit the `exchange_server.d/conf.yaml` file, in the `conf.d/` folder at the root of your Agent's configuration directory to start collecting your Exchange Server performance data.
-    See the [sample exchange_server.d/conf.yaml][2] for all available configuration options.
+1. Edit the `exchange_server.d/conf.yaml` file, in the `conf.d/` folder at the root of your Agent's configuration directory to start collecting your Exchange Server performance data.  
+```yaml
```

Comment:
Sorry, just noticed, but this block of code should be indented, so it appears under the list item. Otherwise it break the numbered list, and we have 
```
1. Edit ...
code
1. Restart the agent
```